modules/arrow/src/geoarrow/convert-geoarrow-to-binary-geometry.ts

```diff
@@ -32,33 +36,41 @@ export const BINARY_GEOMETRY_TEMPLATE = {
   featureIds: {value: new Uint32Array(0), size: 1}
 };
 
+export type BinaryGeometriesFromArrowOptions = {
+  // specify which chunk to get binary geometries from, for progressive rendering
```

Comment:
Nit: Use `/** */` syntax instead and these comments will show up in vscode when we hover over fields.